src/components/ReviewsContainer/ButtonComments.jsx

```diff
@@ -1,8 +1,15 @@
+import { useNavigate } from "react-router"
+
+const ButtonComments = ({reviewID}) => {
+    
+    const navigate = useNavigate()
+    function singleReviewPage(){
+        navigate(`/review/${reviewID}`)
+    }
 
-const ButtonComments = () => {
     return(
         <div>
-            <button>Comments</button>
+            <button onClick={singleReviewPage}>Comments</button>
         </div>
     )
```

Comment:
i don't think you should be using `navigate` here or `onClick` 
we have access to `Routes` and `Link` so why not use them? 
also having to click the `comments` button to view the individual review is not very user friendly and it took me a while to figure out they were even clickable.